npm/rules.bzl

```diff
@@ -17,121 +17,8 @@
 # under the License.
```

Comment:
This file no longer contains any rule implementations itself, but rather re-exports the implementations in the subpackages, helping to keep the codebase clean / adherent to single responsibility principle.